### ISSUE TITLE
 Touch based gestures used for views

### ISSUE THREAD

**Issue by DanteAndroid** [nisrulz/sensey#23]
Can Touch-based gestures be used for views?
I didn't see a view passed into related methods.

Besides, the naming is elegant, it's better if gestures' names are explicated. (e.g. I'm confused with `Wave` and `Shake`)

Nice work, BTW.

**Comment by nisrulz** [comment]
@DanteAndroid you might wanna check out issue #13. Let me know if it solves your issue.

**Comment by DanteAndroid** [comment]
I don't think it's a nice way to use a Custom View. If Touch-based gestures can't be used for views, what is its meaning? Just for Activity?
It'd be great if you could improve it when you have time:)

**Comment by nisrulz** [comment]
@DanteAndroid I agree and I will look into it. For now, the current version only supports activity or custom view. Keeping this issue open until fixed.
